### PR TITLE
Add search filter by price field

### PR DESCRIPTION
**Pull request by Yontih**
Now able to search only on free/paid/both apps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ gplay.list({
     num: 2
   })
   .then(function(apps){
-    console.log('Retrieved ' + apps.length + ' applications!');
+    console.log('Retrieved ' + apps.length + ' applications!');§
```
Comment (facundoolano) [inline README.md]: Typo?

Comment (Yontih) [inline README.md]: Fixed

**Comment by facundoolano** [comment]
Could you change the value of the filters to readable strings like "free", "paid", "all" and convert those to the integers google expects?

**Comment by Yontih** [comment]
done

**Comment by facundoolano** [comment]
Thanks!

**Comment by Yontih** [comment]
Thank you for the quick response!
When this feature will be submitted to the npm module?

**Comment by facundoolano** [comment]
released a new version just now, have fun!

**Pull request merged**